tests/Fixture/FileStorageFixture.php

```diff
@@ -32,13 +32,14 @@ class FileStorageFixture extends TestFixture
         'user_id' => ['type' => 'string', 'null' => true, 'default' => null, 'length' => 36],
         'foreign_key' => ['type' => 'string', 'null' => true, 'default' => null, 'length' => 36],
```

Comment:
We probably want to add another fixture for AIIDs.

Reply:
OK